test/JDBC/input/BABEL-SP_SPROC_COLUMNS.sql

```diff
@@ -1,255 +0,0 @@
-DROP FUNCTION IF EXISTS net
```

Comment:
Removed this file (along with the `.out` version in the `expected` folder) because the three version upgrade tests:
- BABEL-SP_SPROC_COLUMNS-vu-prepare.sql
- BABEL-SP_SPROC_COLUMNS-vu-verify.sql
- BABEL-SP_SPROC_COLUMNS-vu-cleanup.sql
covers all these tests (plus the newly added one). Keeping this file in would cause the JDBC tests to run this test twice.